Assets/AppCenter/Plugins/AppCenterSDK/Auth/Android/AuthInternal.cs

```diff
@@ -31,6 +31,31 @@ public static void AddNativeType(List<IntPtr> nativeTypes)
             nativeTypes.Add(_auth.GetRawClass());
         }
 
+        public static AppCenterTask<UserInformation> SignInAsync()
+        {
+            var future = _auth.CallStatic<AndroidJavaObject>("signIn");
+            var javaTask = new AppCenterTask<AndroidJavaObject>(future);
```

Comment:
This additional wrapper isn't needed

Reply:
I tried but looks like only `AppCenterTask` has the `ContinueWith` method

Reply:
This is an async method that returns a task for UserInformation. It should get SignInResult and return UserInformation.